### PR TITLE
Update system_message.txt

### PR DESCRIPTION
**Pull request by mrfixit-stickyhash**
Trying to stop it from running the same code over and over

### CODE REVIEW

File: interpreter/system_message.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ When you send a message containing code to run_code, it will be executed **on th
 Never use (!) when running commands.
 Only use the function you have been provided with, run_code.
 If you want to send data between programming languages, save the data to a txt or json.
-You can access the internet. Run **any code** to achieve the goal, and if at first you don't succeed, try again and again.
+You can access the internet. Run **any code** to achieve the goal, and if at first you don't succeed, try again A different way.
```
Comment (ericrallen) [inline interpreter/system_message.txt:7]: After the refactor, I think this needs to go in [`interpreter/config.yaml`](https://github.com/KillianLucas/open-interpreter/blob/main/interpreter/config.yaml) to be up to date, but I also haven't noticed it trying the same action over and over - it always seems to tweak it's attempt based on any provided error traces.

It's also worth noting that user's can customize their `system_prompt` now, so this is something that can be added to each user's own personal [`config.yaml`](https://github.com/KillianLucas/open-interpreter/tree/main#configuration) if they find it necessary.

Comment (mrfixit-stickyhash) [reply interpreter/system_message.txt:7]: I don't think this is a problem for gpt But for llama it would continue the same action over and over

Comment (mrfixit-stickyhash) [reply interpreter/system_message.txt:7]: I see your point though ericrallen I'll close the pr if we're switching to mistral but it's been >2 hours setting up the language model update im Currently at 91%

**Comment by ericrallen** [comment]
If it’s Llama-specific, you might want to take a look at some of the manipulating that Open Interpreter had to do to get it to cooperate, and consider adding it there. It seem like the System Message might be less important to Llama.

https://github.com/KillianLucas/open-interpreter/blob/main/interpreter/llm/setup_local_text_llm.py#L275C15-L275C15

**Comment by ericrallen** [comment]
Going to close this for now as the `system_message` control provided in more recent versions should allow this to be done on a per-user (or per-model using the `--config_file` functionality) basis.

**Pull request closed without merging**